client_src/js/services/local-storage.service.js

```diff
@@ -0,0 +1,15 @@
+import { ApplicationError } from "../exceptions/application-error.handler";
+import { ClientErrors } from "../exceptions/octofarm-client.exceptions";
+
+export function removeLocalStorage(key) {
+  if (!!key) throw new ApplicationError(ClientErrors.FAILED_VALIDATION_KEY);
+  const serializedData = getLocalStorage(key);
+  if (!!serializedData && serializedData.length !== 0) {
+    localStorage.removeItem("dashboardConfiguration");
```

Comment:
What is happening here, how did the dashboardConfiguration come into the story?

Reply:
Whoops good spot, fixed.